src/frontend/App.js

```diff
@@ -2,17 +2,17 @@ import React from 'react';
 import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
 import Results from './components/Result/Results';
 import Profile from './components/Profile/Profile';
-import RoutingPage from './components/RoutingPage/RoutingPage';
+import Home from './components/Home/Home';
+import Header from './components/Header/Header';
 import './App.css';
-import Header from '../frontend/components/Header/Header'
 
 export default function App() {
```

Comment:
I cant get my head around "Request for pair programming" been split into two separate h2 tags? otherwise  looks good!